merlin/models/tf/core/tabular.py

```diff
@@ -140,6 +141,10 @@ def __init__(
         if schema:
             self.set_schema(schema)
 
+    def select_by_tag(self, tags: Tags) -> Optional["TabularBlock"]:
```

Comment:
Is this method required if the `Block` has `select_by_tag` implemented? Or do we need to do something different here?

Reply:
We need a new method here because need to sub-select the columns we want.